packages/open-next/src/build.ts

```diff
@@ -489,6 +489,25 @@ async function createServerBundle(monorepoRoot: string) {
   injectMiddlewareGeolocation(outputPath, packagePath);
   removeCachedPages(outputPath, packagePath);
   addCacheHandler(outputPath);
+
+  if (options.minify) {
+    removeNodeModule(path.join(outputPath, "node_modules"), [
+      "@esbuild",
+      "prisma/libquery_engine-darwin-arm64.dylib.node",
```

Comment:
can we include all prisma engines?

Reply:
Do you know where I can get a list of all non-lambda engines?

Reply:
I have some generic prisma node_modules cleanup code https://github.com/jetbridge/sst-prisma/blob/master/stacks/resources/prismaLayer.ts#L90

Reply:
Honestly, what the hell is wrong with prisma?